src/frequenz/client/dispatch/_client.py

```diff
@@ -49,23 +49,34 @@
 # pylint: enable=no-name-in-module
 DEFAULT_DISPATCH_PORT = 443
 
+DEFAULT_GRPC_CALL_TIMEOUT = timedelta(seconds=60)
+"""Default gRPC call timeout."""
+
+DEFAULT_GRPC_STREAM_TIMEOUT = timedelta(minutes=5)
+"""Default gRPC stream timeout."""
+
 
 class DispatchApiClient(BaseApiClient[dispatch_pb2_grpc.MicrogridDispatchServiceStub]):
     """Dispatch API client."""
 
+    # pylint: disable-next=too-many-arguments
     def __init__(
         self,
         *,
         server_url: str,
         key: str,
         connect: bool = True,
+        call_timeout: timedelta | None = None,
+        stream_timeout: timedelta | None = None,
```

Comment:
Why not
```suggestion
        call_timeout: timedelta = timedelta(seconds=60)
        stream_timeout: timedelta = timedelta(minutes=5),
```
and remove the `DEFAULT_XXX` (and the defaults from the docs, the defaults will be shown in the docs anyway like this).

Reply:
I thought we want to avoid to init objects in the default parameters as they aren't constants.
Granted, as we only ever read this value it doesn't matter too much here..

Reply:
`datetime` and `timedelta` are immutable (like a `frozen` dataclass`).